test/controllers/gems_adoptions_controller_test.rb

```diff
@@ -31,10 +31,17 @@ def setup
     assert_redirected_to gem_path(id: 1)
   end
 
+  test "should close a gem up for adoption" do
+    adoption_requests = adoption_requests(:one)
+    put :update, id: adoption_requests.id
+    #assert_equal 'close', adoption_requests.gems_adoption.status
```

Comment:
We also probably want to check that the `GemOwnershipTransfer` for the adoption request in the test was created. What do you think?